client/cs_proxy.cpp

```diff
@@ -76,7 +76,7 @@ void select_proxy_info() {
         working_proxy_info = config_proxy_info;
         msg_printf(0, MSG_INFO, "Using proxy info from cc_config.xml");
         if (env_var_proxy_info.present) {
-            msg_printf(0, MSG_INFO, "Proxy info env vars overridden by GUI");
+            msg_printf(0, MSG_INFO, "Proxy info env vars overridden by cc_config.xml");
```

Comment:
Line 79 is only executed if the proxy settings are made via cc_config.xml and a proxy is **not** set via GUI.
The GUI settings are reported by the code in lines 66-74.